grid_map_demos/CMakeLists.txt

```diff
@@ -82,143 +71,184 @@ add_executable(
   src/ImageToGridmapDemo.cpp
 )
 
-add_executable(
-  octomap_to_gridmap_demo
-  src/octomap_to_gridmap_demo_node.cpp
-  src/OctomapToGridmapDemo.cpp
-)
-
-add_executable(
-  move_demo
-  src/move_demo_node.cpp
-)
-
-add_executable(
-  iterator_benchmark
-  src/iterator_benchmark.cpp
-)
-
-add_executable(
-  opencv_demo
-  src/opencv_demo_node.cpp
-)
-
-add_executable(
-  resolution_change_demo
-  src/resolution_change_demo_node.cpp
-)
-
-add_executable(
-  filters_demo
-  src/filters_demo_node.cpp
-  src/FiltersDemo.cpp
-)
-
-add_executable(
-  normal_filter_comparison_demo
-  src/normal_filter_comparison_node.cpp
-)
-
-add_executable(
-  interpolation_demo
-  src/interpolation_demo_node.cpp
-  src/InterpolationDemo.cpp
-)
+# add_executable(
+#   octomap_to_gridmap_demo
+#   src/octomap_to_gridmap_demo_node.cpp
+#   src/OctomapToGridmapDemo.cpp
+# )
+
+# add_executable(
+#   move_demo
+#   src/move_demo_node.cpp
+# )
+
+# add_executable(
+#   iterator_benchmark
+#   src/iterator_benchmark.cpp
+# )
+
+# add_executable(
+#   opencv_demo
+#   src/opencv_demo_node.cpp
+# )
+
+# add_executable(
+#   resolution_change_demo
+#   src/resolution_change_demo_node.cpp
+# )
+
+# add_executable(
+#   filters_demo
+#   src/filters_demo_node.cpp
+#   src/FiltersDemo.cpp
+# )
+
+# add_executable(
+#   normal_filter_comparison_demo
+#   src/normal_filter_comparison_node.cpp
+# )
+
+# add_executable(
+#   interpolation_demo
+#   src/interpolation_demo_node.cpp
+#   src/InterpolationDemo.cpp
+# )
 
 ## Specify libraries to link a library or executable target against
-target_link_libraries(
+ament_target_dependencies(
   simple_demo
-  ${catkin_LIBRARIES}
+  "rclcpp"
```

Comment:
Use `dependencies` these are all the same

Reply:
I was planning on using `dependencies` once I all the nodes are ported to know all the common dependencies. So this should done in an upcoming PR, is that okay.